crates/cxx-qt-gen/src/generator/naming/signals.rs

```diff
@@ -25,6 +25,17 @@ impl From<&ParsedSignal> for QSignalName {
     }
 }
 
+impl QSignalName {
+    pub fn free_namespace_from_qobject_namespace(qobject_namespace: Option<&String>) -> String {
```

Comment:
Could you move this up into the `naming/mod.rs` file?
As it doesn't really have anything to do with QSignal, but is more generally usable as the "private" namespace for our generation.